shared_workflow/install_shared.py

```diff
@@ -97,7 +97,10 @@ def install_simulation(
         FaultParams.root_yaml_path.value: root_yaml_path,
         FaultParams.vel_mod_dir.value: vel_mod_dir,
     }
-
+    # read VM params
```

Comment:
yeah, some of the changes should be break into a different PR, instead of being in this huge one.
i'm cleaning it up and will move less related ones into separate PR